kite-data/kite-data-flume/pom.xml

```diff
@@ -105,6 +105,12 @@
       <version>1.0.1-SNAPSHOT</version>
     </dependency>
     <dependency>
+      <groupId>org.kitesdk</groupId>
```

Comment:
Flume relies on StorageKey (SPI), which now exposes a Path. That was causing the tests to fail because the Hadoop dependency wasn't present. That was because this is provided everywhere and isn't ever a transitive dep.